crates/syntax/src/ast/make.rs

```diff
@@ -158,6 +158,49 @@ fn ty_from_text(text: &str) -> ast::Type {
     ast_from_text(&format!("type _T = {text};"))
 }
 
+/// Related goto [link](https://doc.rust-lang.org/reference/items/type-aliases.html)
+///     Type Alias syntax is
+///
+///     ```
+///     TypeAlias :
+///        type IDENTIFIER GenericParams? ( : TypeParamBounds )? WhereClause? ( = Type WhereClause?)? ;
+///     ```
+///
+///     FIXME : ident should be of type ast::Ident
```

Comment:
The indentations should be fixed up as well here (for /// doc comments, its idiomatic to have one space between the /// and the text content of the comment